src/library/dao/playlistdao.cpp

```diff
@@ -331,6 +332,33 @@ unsigned int PlaylistDAO::playlistCount() const {
     return numRecords;
 }
 
+QList<QPair<int, QString>> PlaylistDAO::getPlaylists(const uint hidden) const {
+    //qDebug() << "PlaylistDAO::getPlaylists(hidden =" << hidden
+    //         << QThread::currentThread() << m_database.connectionName();
+
+    QSqlQuery query(m_database);
+    query.prepare(
+            mixxx::DbConnection::collateLexicographically(QString(
+                    "SELECT id, name FROM Playlists "
+                    "WHERE hidden = %1 "
+                    "ORDER BY name")
+                                                                  .arg(hidden)));
```

Comment:
https://github.com/llvm/llvm-project/issues/58708

Reply:
thanks.
Since those issues occur every now and then with clang-format I'll need to find a way Eclipse applies to correct indentation in the first place.